src/components/dictionaryConcepts/components/ActionButtons.jsx

```diff
@@ -12,7 +12,7 @@ const ActionButtons = ({
       {actionButtons === true &&
       <Link
         className="btn btn-sm mb-1 actionButtons"
-        to={`${dictionaryPathName}/edit/${id}/${concept_class}`}
+        to={`/edit${dictionaryPathName}/${id}/${concept_class}`}
```

Comment:
There should be a back slash after edit?

Reply:
Yes please @kmunene

Reply:
There shouldn't be. The backslash is already there in the pathname.

Reply:
@dondrzzy @alexmochu if the backslash is added there will be two backslashes.